clang/lib/CIR/CodeGen/CIRGenAtomic.cpp

```diff
@@ -507,9 +509,11 @@ RValue CIRGenFunction::emitAtomicExpr(AtomicExpr *e) {
   }
 
   bool isStore = e->getOp() == AtomicExpr::AO__c11_atomic_store ||
-                 e->getOp() == AtomicExpr::AO__atomic_store;
```

Comment:
It's probably better to add the full set of loads and stores for `isStore` or `isLoad` already - these are very easy to miss (which could make the state have both true or false, which might get tricky to debug) and they actual implementation are already NYI before this path is hit anyways